packagers/jam_test.go

```diff
@@ -16,12 +19,22 @@ func testJam(t *testing.T, context spec.G, it spec.S) {
 		Expect = NewWithT(t).Expect
 
 		executable *fakes.Executable
-		packager   packagers.Jam
+		pack       *fakes.Executable
+
+		tempOutput func(string, string) (string, error)
+
+		packager packagers.Jam
 	)
 
 	it.Before(func() {
 		executable = &fakes.Executable{}
-		packager = packagers.NewJam().WithExecutable(executable)
+		pack = &fakes.Executable{}
+
+		tempOutput = func(string, string) (string, error) {
+			return "some-jam-output", nil
```

Comment:
Does this work if the tests are run in parallel?

I was expecting to see `t.TempDir` as a testing substitute for the default `os.MkdirTemp`

Reply:
I don't think I have ever seen issues with `os.MkdirTemp` and thread safety. It was just the function that I am most familiar with.